core/templates/pages/skill-editor-page/editor-tab/skill-concept-card-editor/skill-concept-card-editor.component.ts

```diff
@@ -149,6 +143,7 @@ angular.module('oppia').component('skillConceptCardEditor', {
             $scope.skill, newExample);
           $scope.bindableFieldsDict.displayedWorkedExamples =
             $scope.skill.getConceptCard().getWorkedExamples();
+          $rootScope.$apply();
```

Comment:
Could you please explain why this is required? Thanks!

Reply:
Whenever we are deleting worked-example its not get deleted in one go (due to change detection issue face by hybrid state), so we have to manually apply it using $rootScope.$apply();

Reply:
This will not be needed once skill-concept-card-editor.component.ts is migrated

Reply:
Sounds good, can you please add a TODO (similar to https://github.com/oppia/oppia/blob/develop/core/templates/components/question-directives/question-player/question-player.component.ts#L581-L582)? Thanks!

Reply:
Sure, will do it in a moment. Thanks

Reply:
Done